Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -153,6 +153,12 @@ static SessionResult webKitMediaOpenCDMDecryptorResetSessionFromKeyIdIfNeeded(We
     LockHolder locker(priv->m_mutex);
     RefPtr<WebCore::CDMInstance> cdmInstance = webKitMediaCommonEncryptionDecryptCDMInstance(self);
     ASSERT(cdmInstance && is<WebCore::CDMInstanceOpenCDM>(*cdmInstance));
+
+    if(!cdmInstance){
```

Comment:
There is an assert before this, so please, let's adapt this into `if (!cdmInstance || !is<WebCore::CDMInstanceOpenCDM>(*cdmInstance))`. Then remove the assert before and move it to inside the if in form of `ASSERT_NOT_REACHED()`.

Reply:
Before accepting this patch or its corrected version, I'm trying to understand why the CDMInstance isn't available on time.